build.sh

```diff
@@ -73,7 +73,13 @@ for WORKER in "${WORKER_DIRS[@]}"; do
   # Compile the schema
   OUT_DIR=improbable/generated
   mkdir -p "$OUT_DIR"
-  "$DOWNLOAD_DIR"/schema_compiler --schema_path="${BUILD_DIR}/SpatialOS/schema" --schema_path="$DOWNLOAD_DIR" --csharp_out="$OUT_DIR" --load_all_schema_on_schema_path "${BUILD_DIR}"/SpatialOS/schema/*.schema "${DOWNLOAD_DIR}"/improbable/*.schema
+  "$DOWNLOAD_DIR"/schema_compiler \
```

Comment:
Instead of generating the same csharp in a loop for every worker, this could be combined with the schema description generator.
Generate both the descriptor and csharp in 1 go, then copy the csharp to each worker.

Reply:
No longer generating the csharp in a loop.  However, have still kept the csharp and schema descriptor generation in two discrete steps to make it seem less magical.